src/foam/nanos/bench/BenchmarkRunner.js

```diff
@@ -182,7 +194,19 @@ foam.CLASS({
       name: 'runScript',
       javaCode: `
       long startTime = System.currentTimeMillis();
+      Logger logger = (Logger) x.get("logger");
+      LogLevelFilterLogger loggerFilter = null;
+      while ( true ) {
+        if ( logger instanceof LogLevelFilterLogger ) {
```

Comment:
this loop makes assumptions about type checking and if we'll ever be able to find a LogLevelFilterLogger. this is debatable, but if the logger reliably behaves this way, we can leave this alone

Reply:
Yes, lots of assumptions. If the above is not true, then the system is broken in general.